prime-dotnet-webapi/Engines/AgreementEngine Internal/CertificationDigest.cs

```diff
@@ -106,6 +113,28 @@ public Pharmacist(bool regulated)
         }
     }
 
+    public class PharmacyTechnician : ICertificationDigest
+    {
+        private bool Regulated { get; set; }
+
+        public PharmacyTechnician(bool regulated)
+        {
+            Regulated = regulated;
+        }
+
+        public AgreementType? ResolveWith(SettingsDigest settings)
+        {
+            if (Regulated)
+            {
+                return AgreementType.PharmacyTechnicianTOA;
+            }
+            else
+            {
+                return null;
```

Comment:
So this should probably give you the obo pharmacy toa not null, it is a very small window but this will send a pharm tech before feb 1st to manual and not the obo toa